cmd/mcp-server/eval/scripts/score-results.py

```
@@ -0,0 +1,390 @@
+                            for c in config_names))
+    lines.append("")
+
+    # Per-scenario details
+    lines.append("## Per-Scenario Results\n")
+    for scenario in scored_results["scenarios"]:
+        lines.append("### %s\n" % scenario["scenario_id"])
+        lines.append("| Metric | %s |" % " | ".join(config_names))
+        lines.append("|--------|%s|" % "|".join(["--------"] * len(config_names)))
+
+        def _get(config, field):
+            data = scenario["configs"].get(config, {})
```

Ruff (0.15.15)

[warning] Function definition does not bind loop variable `scenario`

(B023)